src/containers/About.js

```diff
@@ -0,0 +1,29 @@
+import '../components/homepage/Homepage.css'
```

Comment:
is there a reason we are importing this css here @nataliyah123?
I don't think we need this, but you have more context than I do so if there is one, could you maybe explain it?